### ISSUE TITLE
Errors: Failed to load resource/history.state is not available in packaged apps

### ISSUE THREAD

**Issue by thoatswold** [e2email-org/e2email#38]
I followed the instructions to build the app on a Ubuntu 16.04 Desktop machine running the latest versions of git, python, ant and Oracle JDK. The build completed without errors.

However, on loading the app as an extension within Chrome, it never gets further than the opening screen, and the developer console shows five errors (three distinct errors) – see screenshots. I have tried this on Ubuntu and also on Windows 10 (using the folder from Ubuntu, as the app won't build on Windows using the git-bash shell), and the result is the same.

![e2email-errors](https://cloud.githubusercontent.com/assets/11094776/23412369/3a6e6ec6-fdcd-11e6-9294-0ebfa689607c.jpg)
![e2email](https://cloud.githubusercontent.com/assets/11094776/23412370/3a7150c8-fdcd-11e6-8be4-a955e189f19a.jpg)

**Comment by kbsriram** [comment]
Is there any chance the app window is cutoff at the bottom? There should be a button below the text that says "... using your Gmail account." Could it be that the default size of the app window is not large enough in your environment to show all of the content?

**Comment by thoatswold** [comment]
Yes, you're right – that was it, thank you! I'm using an oldish laptop with
a 1366 x 768 screen, so the vertical height wasn't enough for the initial
content. I rotated the screen and completed the setup, then reverted to
normal and it was fine.

I suggest that the window should have a vertical scrollbar to allow for the
possibility of a low-res screen. Also the errors I found, although not
fatal, should perhaps be treated as bugs.
